frontend/app/src/services/reports/const.ts

```diff
@@ -0,0 +1 @@
+export const reportNumericKeys = ['size_on_disk'];
```

Comment:
Since we are moving to zod, if you use the `NumericString` type this will not be needed. You can leave the `numericFields` property as an empty array to avoid the conversion both at the axios level and when zod is parsing the data.